logservice_test.go

```diff
@@ -68,6 +69,17 @@ func (m *mockSDStoreUploader) Upload(path string, filePath string) error {
 	return nil
 }
 
+type mockScrewdriverAPI struct {
+	updateSteplines func(string, int) error
```

Comment:
for consistency, also should have everything be camel case for `updateStepLines`